cypress/e2e/GUI/disables_input_field_no_customers_in_db.cy.js

```diff
@@ -0,0 +1,12 @@
+describe('Desativação do campo de entrada de texto quando não há clientes', () => {
+    beforeEach(() => {
+        cy.intercept('GET', 'http://localhost:3001/customers?page=1&limit=10&size=All', { statusCode: 200, body: `{customers:[]}` }).as('getCustomers');
+    });
+
+    it('Deve desativar o campo de entrada de texto quando não há clientes', () => {
+      cy.visit('/clientes');
```

Comment:
Não existe a URL `/clientes`.

```suggestion
      cy.visit('/');
```